packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -81,6 +82,19 @@ export async function createPatient({
 
   const newPatient = await PatientModel.create(patientCreate);
 
+  analytics({
```

Comment:
Patient creates dont always happen from the public endpoint so adding this here to track other instances such as bulk import